inference-engine/src/cldnn_engine/cldnn_primitives_list.hpp

```diff
@@ -88,7 +88,7 @@ REGISTER_FACTORY(v0, Unsqueeze);
 // REGISTER_FACTORY(v0, Range);
 // REGISTER_FACTORY(v0, RNNCell);
 // REGISTER_FACTORY(v0, ShapeOf);
-// REGISTER_FACTORY(v0, TensorIterator);
+REGISTER_FACTORY(v0, TensorIterator);
```

Comment:
What about Loop-5 op support? Will this primitive support Loop operation with const iterations count?

Reply:
Technically that will be supported with fixed lenght once we implemented CreateLoopOp in cldnn_engine and validated. However this PR's scope is to enable TensorIteratorOp.